android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1207,6 +1207,21 @@ private void applyCustomBackground(boolean reuseCurrentDrawable)
 		}
 	}
 
+	private void setClipMode(int clipMode)
+	{
+
+		if (nativeView instanceof ViewGroup viewGroup) {
+			if (clipMode == UIModule.CLIP_MODE_DISABLED) {
```

Comment:
Just the `setClipChildren(clipMode != UIModule.CLIP_MODE_DISABLED)` is fine. Further we can internally set this on top-most view as discussed earlier.

Reply:
Yes, thanks for that :+1: 

I've put it into draft as I know that parts are still missing especially the parent part. I've added your other suggestions. Thanks for the review :+1: